client/output.go

```diff
@@ -0,0 +1,34 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/spf13/viper"
+	"github.com/tendermint/tendermint/libs/cli"
+)
+
+// Printable defines which structs can be printed by
+// CLI output functions
+type Printable interface {
```

Comment:
I don't see why we're implementing a new interface. This is what `Stringer` is for 👍

Reply:
Duh, I'll make that fix. Will be easier/